kinesis.go

```diff
@@ -55,6 +55,7 @@ func (a *KinesisAdapter) Stream(logstream chan *router.Message) {
 		}
 
 		if streamName == "" {
+			debugLog("The stream name is empty, couldn't match the template. Skipping the log.\n")
```

Comment:
I have no way to check for a typo, unless: 1) the template compilation crash, if which case you'll get alerted: https://github.com/remind101/logspout-kinesis/blob/master/kinesis.go#L31, or 2) the result of executing the template against router.Message is an empty string (probably not something anyone want). But it's it results in anything else, maybe that's what this person want.